File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    int space;
    int i, j;
    int k = 0;
    int n, m;

    for (i = 1; i <= height; i++, k = 0, m = 0)
    {
        for (space = 0; space < height - i; space++)
        {
            printf(" ");
        }
        while (k != i)
        {
            printf("#");
            k++;
        }
        printf("  ");

        while (m != i)
        {
            printf("#");
            m++;
        }
        printf("\n");
    }



}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break
k = 1
j = 1

for i in range(height - 1, -1, -1):
    print(" " * i, end="")
    print("#" * k, end="")
    print("  ", end="")
    print("#" * j, end="")
    k += 1
    j += 1
    print()


